packages/docusaurus/src/webpack/plugins/LogPlugin.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import WebpackBar from 'webpackbar';
+import formatWebpackMessages from 'react-dev-utils/formatWebpackMessages';
+import type {Compiler, Plugin} from 'webpack';
+
+class LogPlugin extends WebpackBar implements Plugin {
+  apply(compiler: Compiler): void {
```

Comment:
`webpackbar` defines apply as `apply(compiler: any): void;` and i had to override type in here